{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:00:41"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:00:41"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:00:41"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:00:43"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:00:43"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:00:43"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:01:45"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:01:45"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:01:45"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:02:07"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:02:07"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:02:07"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:03:17"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:03:17"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:03:17"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:04:36"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:04:36"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:04:36"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:06:13"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:06:13"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:06:13"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:10:10"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:10:10"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:10:10"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:10:20"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:10:20"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:10:20"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:10:23"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:10:23"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:10:23"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:11:53"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:11:53"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:11:53"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:11:57"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:11:57"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:11:57"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:12:10"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:12:10"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:12:10"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:12:12"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:12:12"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:12:12"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:16:00"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:16:00"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:16:00"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:16:04"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:16:04"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:16:04"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:16:06"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:16:06"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:16:06"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:17:16"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:17:17"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:17:17"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:17:21"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:17:21"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:17:21"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:17:57"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:17:57"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:17:57"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:18:55"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:18:55"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:18:55"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:19:02"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:19:02"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:19:02"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:20:57"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:20:57"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:20:57"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:20:59"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:20:59"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:20:59"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:22:04"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:22:05"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:22:05"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:22:25"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:22:25"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:22:25"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:23:03"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:23:03"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:23:03"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:24:03"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:24:03"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:24:03"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:24:18"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:24:18"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:24:18"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:25:23"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:25:23"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:25:23"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:31:19"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:31:20"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:31:20"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:32:05"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:32:05"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:32:05"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:36:54"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:36:54"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:36:54"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:38:33"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:38:33"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:38:33"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:38:36"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:38:36"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:38:36"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:38:49"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:38:49"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:38:49"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:39:35"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:39:35"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:39:35"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:39:53"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:39:53"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:39:53"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:40:18"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:40:18"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:40:18"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:40:35"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:40:35"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:40:35"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:41:53"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:41:53"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:41:53"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:42:09"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:42:09"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:42:09"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:42:55"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:42:55"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:42:55"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:43:06"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:43:06"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:43:06"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:43:14"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:43:15"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:43:15"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:43:28"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:43:28"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:43:28"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:43:50"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:43:50"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:43:50"}
{"message":"development, Picaso.io App listening on port 8080","level":"info","timestamp":"2019-02-17 20:44:05"}
{"message":"MONGODB connection errorMongoNetworkError: failed to connect to server [localhost:27017] on first connect [MongoNetworkError: connect ECONNREFUSED 127.0.0.1:27017]","level":"error","timestamp":"2019-02-17 20:44:05"}
{"message":"MONGODB connection error","level":"error","timestamp":"2019-02-17 20:44:05"}
